fronts-client/src/services/pandaFetch.ts

```diff
@@ -1,6 +1,8 @@
 import { reEstablishSession } from 'panda-session';
+import notifications from './notifications';
 
 const reauthUrl = '/login/status';
+const reauthErrorMessage = `We couldn't log you back in. Your changes may not be saved. Please log in again to reauthenticate.`;
```

Comment:
Could we stick a link in here, so users can click through to log in again?

Reply:
Yes, we can! Excellent idea, thanks for the suggestion @jennygrahamjones 👍